getdescriptives/R/get-descriptives.R

```diff
@@ -333,7 +335,11 @@ assignInNamespace("wt", wtOrdinal, ns = "arsenal")
   # it will parse them to obtain the demographics dataframe
   if (class(userInput) == "numeric") {
     return(.demographicsDf(userInput))
-  } else if (class(userInput) == "data.frame") {
+  } else if ('tbl_df' %in% class(userInput)) {
+    return(as.data.frame(userInput))
+  } else if ('tbl'  %in% class(userInput)) {
+    return(as.data.frame(userInput))
+  } else if ('data.frame'  %in% class(starwars)) {
```

Comment:
Here we shouldn't be using class(starwars) as this will be always be true, moreover starwars is not a parameter ig the function checkInput